query/raw_result.go

```diff
@@ -192,6 +192,7 @@ func (h RawResultHeap) heapify(i, n int) {
 }
 
 // RawResults is a collection of raw results.
+// TODO(wjang): add JSON marshaling / unmarshaling
```

Comment:
Comments start w/ upper case and should end w/ a period as they are complete sentences.